src/ResourceManagement/SiteRecovery/SiteRecovery.Tests/ScenarioTests/SiteTests.cs

```diff
@@ -60,6 +62,40 @@ public void DeleteSite()
             }
         }
 
+        [Fact]
+        public void A2ACreateSite()
+        {
+            using (UndoContext context = UndoContext.Current)
+            {
+                context.Start();
+                var client = GetSiteRecoveryClient(CustomHttpHandler, Constants.A2A);
+
+                FabricCreationInput siteInput = new FabricCreationInput();
+                siteInput.Properties = new FabricCreationInputProperties();
+
+                siteInput.Properties.CustomDetails = new AzureFabricCreationInput()
+                {
+                    Location = location
+                };
+
+                var response = client.Fabrics.BeginCreating(azureSiteName, siteInput, RequestHeaders);
```

Comment:
This flow is testing the async call. So we are not expecting fabric to be created even after the call is a success.